src/main/resources/tailormap-api.yaml

```diff
@@ -730,3 +730,65 @@ paths:
               schema:
                 $ref: './statusresponses.yaml#/components/schemas/ErrorResponse'
 
+  /app/{appId}/layer/{layerId}/proxy/wms:
```

Comment:
this matches only part of the functionality as there is a mapping for `/app/{appId}/layer/{layerId}/proxy/wtms` as well, but that is not documented through the API

I would suggest updating the url with an in-path variable eg. `/app/{appId}/layer/{layerId}/proxy/{protocol}` and updating the other parameters to match or adding a new mapping for wmts only

Alternatively, either remove/disable the wmts proxy as https://b3partners.atlassian.net/browse/HTM-475?focusedCommentId=39665 states it is not required or remove the endpoint from the API as it is not really an API function

Reply:
Updated the API, but keeping WMTS as that is a useful feature